### ISSUE TITLE
Key server URL missing in documentation

### ISSUE THREAD

**Issue by mss1010** [corona-warn-app/cwa-documentation#450]
## Where to find the issue
https://github.com/corona-warn-app/cwa-server/blob/master/docs/DISTRIBUTION.md

## Describe the issue
On the above page, the exposure key distribution is described along with descriptions of the files and directory structure. However, nowhere in the documentation I found the information, what is the actual endpoint URL, where the german CWA server is deployed. I understand, that the documentation is kept very generic, but still this repository contains the GERMAN deployment of the whole solution. Therefore I think it would make sense to mention the actual URLs somewhere.

Or is it already mentioned somewhere, and I just don´t find it?

## Suggested change
Add the actual URL of the german CWA server, which can be used to get the exposure keys, to the documentation

**Comment by MikeMcC399** [comment]
@mss1010 
Your question has gone unanswered for a long time!

I think the information you are looking for is in the https://github.com/corona-warn-app/cwa-documentation/blob/master/backend-infrastructure-architecture.pdf document in Section 3.2.5

**Comment by ndegendogo** [comment]
@MikeMcC399 
1) unfortunately, this document is dated 15 June and does not reflect newer features (daily vs hourly keys, DE vs EUR endpoints for keys).

2) I assume you are referring to table 3. But here entries for getting the DEK keys are missing.

3) Table 2, column DNS Name: last line is given as 'submission coronawarn.app' (typo?)

4) Section 3.2.3.8, last sentence:
> Note: please keep in mind, that the URLs and the implementation is still under development. Changes may occur and may not reflect the actual implementation.

*It seems that this document should be reviewed and updated. Shall I open a separate ticket for these findings, or shall we continue with this ticket?*

**Comment by Ein-Tim** [comment]
@ndegendogo 
IMHO this is a separate ticket

**Comment by MikeMcC399** [comment]
@ndegendogo 
You are right that the document https://github.com/corona-warn-app/cwa-documentation/blob/master/backend-infrastructure-architecture.pdf is out of date. Also it has quite a few inaccuracies in it. Worst of all, it does **not** answer the question about the URL of the key server! (Whoops! I apologise!)

@heinezen said in https://github.com/corona-warn-app/cwa-website/issues/613#issuecomment-735415369 that he was going to try to get the document updated.

Also, I myself noted one month ago in https://github.com/corona-warn-app/cwa-website/issues/613#issuecomment-732940255 that 
`svc90.main.px.t-online.de`
was missing from the document. I forgot that I had discovered this omission! 

[prod_environments.json](https://github.com/corona-warn-app/cwa-app-android/blob/main/prod_environments.json) shows the current code setting for the Android app to be
`"DOWNLOAD_CDN_URL": "https://svc90.main.px.t-online.de",`
so that currently is a reliable source of information.

The other end-point details are in
https://github.com/corona-warn-app/cwa-server/raw/main/services/distribution/api_v1.json
which is referred to in https://github.com/corona-warn-app/cwa-server/blob/main/README.md#service-apis under Distribution Service.

> It seems that this document should be reviewed and updated. Shall I open a separate ticket for these findings, or shall we continue with this ticket?

At the minimum `"DOWNLOAD_CDN_URL": "https://svc90.main.px.t-online.de",` should be added to Table 2 of the document https://github.com/corona-warn-app/cwa-documentation/blob/master/backend-infrastructure-architecture.pdf.

It may make sense to start a new issue for other points which need correcting in the document.

**Comment by ndegendogo** [comment]
I have submitted https://github.com/corona-warn-app/cwa-documentation/issues/496 for a request / suggestion to update the backend infrastructure architecture document.

**Comment by Ein-Tim** [comment]
There was no process on this issue in over a year. The Open-Soruce-Team should decide how to proceed here, either close it with a reference to the more general request https://github.com/corona-warn-app/cwa-documentation/issues/496 or add the URL to the document via PR.
